auth/template.yaml

```diff
@@ -28,6 +28,21 @@ Parameters:
       The ARN of the permissions boundary to apply to any role created by the template
     Default: "none"
 
+  CustomScopeName:
+    Type: String
+    Description: Name for the custom OAuth scope
+    Default: "read"
+
+  CustomScopeDescription:
+    Type: String
+    Description: Description for the custom OAuth scope
+    Default: Generating a token for the client
+
+  ResourceServerName:
+    Type: String
+    Description: Name for the Cognito Resource Server
+    Default: SharedSignalResourceServer
```

Comment:
Do we need these as parameters? I think we may be better off just defining these in the resource definitions? Rather than make them configurable via the `sam deploy`